### ISSUE TITLE
Invalid StackName if target contains space

### ISSUE THREAD

**Issue by PGLongo** [michaelarmstrong/SuperRecord#28]
If the target contains space the model is automatically named with '_' instead of ' ', for example 'Pokemon Ruby Omega' became 'Pokemon_Ruby_Omega'

```swift
let stackName = (infoDictionary!["CFBundleName"] as NSString
```

We could change stackName like this

```swift
let stackName = (infoDictionary!["CFBundleName"] as NSString).stringByReplacingOccurrencesOfString(" ", withString: "_")
```

Maybe we can also change NSString to String.

**Comment by PGLongo** [comment]
Maybe you can fix in #22

**Comment by michaelarmstrong** [comment]
I can confirm this will indeed be fixed in #22
